Makefile

```diff
@@ -1,7 +1,7 @@
 darwin:
 	/bin/bash -c "mkdir -p bin/darwin"
-	/bin/bash -c "crystal build src/scry.cr --release -s --no-debug -o bin/darwin/scry"
```

Comment:
Why remove `--release` from the Makefile? Won't this make final compiled binaries larger and unoptimized?

Reply:
I thought we are putting to many pressure in Travis, `--release` needs a lot of resources :sweat_smile:

`--no-debug` binaries are good enough I think, WDYT?

Reply:
The Makefile is only executed after unit tests are successful and will not be executed if unit tests fail. This build step is building the artifacts we publish to GitHub Releases, hence I believe it important to make the release builds as optimized as possible.

Don't worry about Travis load if it becomes an issue we will be informed. The main crystal repo has builds that more than 30 minutes. Our's have averaged in the low teens.